src/core/gui/toolbarMenubar/icon/ToolbarSeparatorImage.cpp

```diff
@@ -3,25 +3,44 @@
 #include <cairo.h>        // for cairo_create, cairo_destroy, cairo_image_su...
 #include <gdk/gdk.h>      // for gdk_pixbuf_get_from_surface
 #include <glib-object.h>  // for g_object_unref
+#include <util/Color.h>   // for Colors::red, Colors::gray
+#include <util/Util.h>    // for cairo_set_source_rgbi
 
-auto ToolbarSeparatorImage::newImage() -> GtkWidget* {
-    GdkPixbuf* pixbuf = ToolbarSeparatorImage::getNewToolPixbuf();
+constexpr double LINE_WIDTH = 5.;
+constexpr double MARGIN = 3.;
+constexpr double TAIL_SIZE = 5.;
+
+auto ToolbarSeparatorImage::newImage(SeparatorType separator) -> GtkWidget* {
+    GdkPixbuf* pixbuf = ToolbarSeparatorImage::getNewToolPixbuf(separator);
     GtkWidget* w = gtk_image_new_from_pixbuf(pixbuf);
     g_object_unref(pixbuf);
     return w;
 }
 
-auto ToolbarSeparatorImage::getNewToolPixbuf() -> GdkPixbuf* {
+auto ToolbarSeparatorImage::getNewToolPixbuf(SeparatorType separator) -> GdkPixbuf* {
```

Comment:
I guess we'd better make an actual icon here, instead of drawing it with cairo.
The icons would then be theme-dependent: if the background color is the same as Colors::gray, we won't see anything.

Reply:
Thanks for the review! 
I will open an issue about the icons and postpone it to another PR. Maybe someone with better designing skills can create them. The color of the widget background is one thing. The other one is the color of the page, when you drag&drop. So the icon should be clearly visible on any page color.

Reply:
Merging in 24 hours if no objections are raised.